bridges/TheYeteeBridge.php

```diff
@@ -0,0 +1,47 @@
+<?php
+class TheYeteeBridge extends BridgeAbstract {
+	
+	const MAINTAINER = 'Monsieur Poutounours';
+	const NAME = 'TheYetee';
+	const URI = 'https://theyetee.com';
+	const CACHE_TIMEOUT = 14400; // 4 h 
+	const DESCRIPTION = 'Fetch daily shirts from The Yetee';
+	
+	public function collectData(){
+		
+		$html = getSimpleHTMLDOM(self::URI)
+			or returnServerError('Could not request The Yetee.');
+			
+		$div = $html->find('div');
+		foreach($div as $element) {
+			
+			if($element->class === 'hero-col')
+			{
+				//echo $element;
+				
+				$item = array();
+				
+				$title = $element->find('h2',0)->plaintext;
+				$item['title'] = $title;
+				
+				$author = trim($element->find('div[class=credit]',0)->plaintext);
+				$item['author'] = $author;
+				
+				$uri = $element->find('div[class=controls] a',0)->href;
+				$item['uri'] = static::URI.$uri;
+				
+				$content = "<p>".$element->find('section[class=product-listing-info] p',-1)->plaintext."</p>";
+				$photos = $element->find('a[class=js-modaal-gallery] img');
+				foreach($photos as $photo)
+				{
+					$content = $content."<br /><img src='".$photo->src."' />";
```

Comment:
You should also add the images to the enclosures

Reply:
Thanks for your comments.
I am not sure to understand this one :-/ Can you show me an example ?

Reply:
The `$item` variable can take an array of links to media elements, called enclosures. This is used by feed readers to display the medias in a specific format. The way you can do this is as follow: 
```PHP
$item['enclosures'] = array();

foreach($photos as $photo) {
	$content = $content."<br /><img src='" . $photo->src . "';
	$item['enclosures'][] = $photo->src;
}